caps/types.go

```diff
@@ -53,6 +54,13 @@ func (t *BoolFileType) Name() string {
 	return "bool-file"
 }
 
+var boolFileAllowedPathPatterns = []*regexp.Regexp{
+	// The brightness of standard LED class device
+	regexp.MustCompile("^/sys/class/leds/[^/]+/brightness$"),
```

Comment:
Please just drop it. This is a very simple test capability that we can improve at will later. This branch could already be merged by now.